runtime/drivers/druid/model_manager.go

```diff
@@ -0,0 +1,105 @@
+package druid
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/rilldata/rill/runtime/drivers"
+)
+
+type ModelInputProperties struct {
+	Path          string `mapstructure:"path"`
+	Pattern       string `mapstructure:"pattern"`
+	Granularity   string `mapstructure:"gran"`
+	Format        string `mapstructure:"format"`
+	FilePattern   string `mapstructure:"file_pattern"`
+	RetriesPeriod string `mapstructure:"retry_period"`
+	MaxRetries    int    `mapstructure:"max_retries"`
+}
```

Comment:
So these are actually properties for an object store connector, not for the Druid connector. Ideally the name `druid.ModelInputProperties` would be reserved for input properties for Druid (i.e. models where Druid is the input and something else is the output – obviously we don't support that now, but could make sense in the future for e.g. models that export to S3).

Ideally the input properties for GCS would be defined in `gcs.ModelInputProperties` (or maybe `drivers.ObjectStoreModelInputProperties` if shared across multiple object store drivers), but I realize these are quite specific to this Druid driver. So maybe having it be an internal `gcsModelInputProperties` would make more sense.